pages/privacy.tsx

```diff
@@ -1,232 +1,127 @@
-// TODO: fix eslint errors - Remove below comments and fix the linting errors
-/* eslint-disable jsx-a11y/alt-text */
-/* eslint-disable react/no-unescaped-entities */
-/* eslint-disable react/react-in-jsx-scope */
 import Head from 'next/head'
+import React from 'react'
 
 import Footer from '../components/Footer/index'
 
 export default function Privacy() {
   return (
     <div className="min-w-full grid-cols-1 grid-rows-3">
       <Head>
-        <link href="https://fonts.googleapis.com/css?family=Roboto:300" rel="stylesheet" type="text/css" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
do not download font file in every page. Download font only in index.html and apply to body